parlai/core/torch_ranker_agent.py

```diff
@@ -664,7 +663,9 @@ def _build_candidates(self, batch, source, mode):
                 use_cuda=self.use_cuda,
                 fp16friendly=self.fp16,
             )
-            if label_vecs is not None:
+            if label_vecs is not None and not batch.id.startswith(
+                'internal:vqa_v2:PersonalityTeacher'
```

Comment:
i'm not sure we should include this in the actual `torch_ranker_agent` code, perhaps this could be in the model?

Reply:
I agree, I do'nt think this should be in TRA